ons-spark/intro.md

```diff
@@ -1,9 +1,25 @@
 # Introduction
 
-This is the intro page to our book. 
+Our **vision** is to fill the gap between Spark text books, official documentation and online examples as the go-to guide for analysts, data scientists and data engineers using PySpark and sparklyr.
 
-This is who we are and the intension or motivation for the book.
+Our **mission** is to share knowledge and experience across government to harness the potential of big data for [statistics for the public good](https://uksa.statisticsauthority.gov.uk/statistics-for-the-public-good/).
 
-ONS staff should also look at our GitLab pages for organisation-specific information.
+Some of our core **values** are:
+- Openness and transparency
+- Diversity and collaboration
+- Good coding practices
+- Efficiency and value for money
 
-Open to contributions from anyone, across UK Govt and beyond.
\ No newline at end of file
+This book is open to contributions from anyone across UK Govt and beyond.
+
+ONS staff should also look at our GitLab pages for organisation and platform-specific information.
+
+## Further resources
+
+Some examples of other free resources:
+
+- [Learning Spark 2nd edition](https://pages.databricks.com/rs/094-YMS-629/images/LearningSpark2.0.pdf?utm_medium=email&utm_source=databricks&utm_campaign=7014N0000026tqzQAA)
+- [Spark official documentation](https://spark.apache.org/docs/latest/)
+- [PySpark documentation](https://spark.apache.org/docs/latest/api/python/)
+- [sparklyr documentation](https://spark.rstudio.com/)
+- [Spark by examples](https://sparkbyexamples.com/)
```

Comment:
Is this an appropriate place to link through to Arthurs book and the Duck Book?

Reply:
done